src/packages/selection/Selection.tsx

```
@@ -118,7 +123,10 @@ export const Selection: FC<SelectionProps> = (props) => {
     );
     const currentSourceStatus = useSourceStatus(currentSource, defaultNotAvailableMessage);
 
-    const [selectionKind, setSelectionKind] = useState<SelectionKind>("extent");
+    const showSelectionButtons = Boolean(selectionMethods && Array.isArray(selectionMethods) && selectionMethods.length > 1);
```

It seems like these checks are not working properly. For me it crashed when `selectionMethods` is a
 - a empty array
   -  using `extent` as default is fine but I should not crash
 - an array with `length===1`
   - should behave in the same way as a single value

---

Fixed the length===1 issue.
However, I would argue that providing an empty list should be a developer error as an empty list doesn't carry any meaning as it is an object and not explicitly null. In my opinion, the program should crash if the (developer) input is nonsense instead of falling back to some default behaviour. 
But that is personal preference, I like my APIs to crash if I mess things up :D